etc/tendrl/node-agent/node-agent.conf.yaml.sample

```diff
@@ -10,3 +10,16 @@ glusterfs_repo: https://buildlogs.centos.org/centos/7/storage/x86_64/gluster-3.9
 with_internal_profiling: False
 tags:
   - tendrl/node
+plugins:
```

Comment:
We shouldnt be allowing users to change these plugins, it is mandatory for Tendrl to provide this data, please put them in definition file or hardcode it in the configure monitoring atom